doc/changelog/01-added/1333-seminorm.md

```diff
@@ -0,0 +1,4 @@
+- in `ssrnum.v`
+  + lemmas `gtr0_norm_neq0`, and `gtr0_norm_eq0F`
+    (`#1333 <https://github.com/coq/stdlib/pull/1333>`_,
+    by Cyril Cohen).
```

Comment:
The changelog `02-changed` says ` by Alessandro Bruni and Cyril Cohen`. Should it not be the same here, as well as in `03-renamed`?